src/k2/transport/RPCDispatcher.h

```diff
@@ -274,6 +281,8 @@ class RPCDispatcher: public seastar::weakly_referencable<RPCDispatcher> {
     RPCDispatcher(RPCDispatcher&& o) = delete;
     RPCDispatcher& operator=(const RPCDispatcher& o) = delete;
     RPCDispatcher& operator=(RPCDispatcher&& o) = delete;
+
+    ConfigVar<bool> _txUseCrossCoreLoopback{"tx_xcore_loopback"};
```

Comment:
set a default value of true for this: _txUseCrossCoreLoopback{"tx_xcore_loopback", true};

Reply:
Set the default value for _txUseCrossCoreLoopback.